src/VisualStudio/Core/Impl/Options/AbstractOptionPageControl.cs

```diff
@@ -24,6 +24,13 @@ public abstract class AbstractOptionPageControl : UserControl
         private readonly List<BindingExpressionBase> _bindingExpressions = new List<BindingExpressionBase>();
         private readonly List<OptionPageSearchHandler> _searchHandlers = new();
 
+#if DEBUG
+        /// <summary>
+        /// Used for testing only!
+        /// </summary>
+        internal IReadOnlyList<BindingExpressionBase> BindingExpressions => _bindingExpressions.AsReadOnly();
+#endif
```

Comment:
This shouldn't be in `#if DEBUG` since tests can be run in release too. I'll cleanup.